src/util/base.cc

```diff
@@ -104,18 +104,15 @@ bool process(
 
 bool BaseCommand::Impl(
     const std::string& cmd_name,
-    int argc,
```

Comment:
I think the correct solution was to use all options AFTER the positional argument as options for the benchmark test. The rerun the bpo::command_line_parser. As it currently stands i wont be able to use any of the options for main --all, -a, etc.  I would rather put it back before I push.